src/Components/ProvisioningWizard/steps/Pubkeys/PubkeySelect.js

```diff
@@ -73,11 +91,17 @@ const PubkeySelect = ({ setStepValidated }) => {
         selections={selection}
         placeholderText="Select public key..."
         aria-label="Select public key"
-        validated={!isKeySupported && 'error'}
+        {...(pubkeys &&
+          pubkeys.length < total && {
```

Comment:
could we use `hasNextPage` from the [useInfiniteQuery return values](https://tanstack.com/query/v4/docs/react/reference/useInfiniteQuery) for this condition? :thinking:

Reply:
I tried but it didn't seem to work, but I'll try again maybe I missed something